src/commands/scale.ts

```diff
@@ -0,0 +1,84 @@
+import { Flags } from '@oclif/core';
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import Account from '../architect/account/account.entity';
+import AccountUtils from '../architect/account/account.utils';
+import Environment from '../architect/environment/environment.entity';
+import { EnvironmentUtils } from '../architect/environment/environment.utils';
+import BaseCommand from '../base-command';
+import { ComponentVersionSlugUtils, ResourceSlugUtils } from '../dependency-manager/spec/utils/slugs';
+import { ComponentVersion } from './components/versions';
+
+export default class Scale extends BaseCommand {
+  static description = 'Scale a service to a specified number of replicas.';
+
+  static flags = {
+    ...AccountUtils.flags,
+    ...EnvironmentUtils.flags,
+    service: Flags.string({
+      description: 'Name of the service to scale',
+      sensitive: false,
+    }),
+    replicas: Flags.integer({
+      name: 'replicas',
+      description: 'Number of desired service replicas',
+      sensitive: false,
+    }),
+  };
+
+  static args = [{
```

Comment:
It feels odd to me this is in an arg. Why is service a flag and this is not a flag? Would like some input from @tjhiggins since I know this keeps some consistency with other commands, but it does feel weird here.

Reply:
That is an extremely fair question. I'd be curious for thoughts/proposals. @TylerAldrich @ryan-cahill

Reply:
I wonder if the opposite is more natural here - e.g. `architect scale -e prod --component tylerland:latest --replicas 5000 my_api_service`? Or maybe they should just all be flags...

`architect scale -e prod --replicas 5000 --service my_api_sevice tylerland:latest` makes more sense if we allow them to omit the service and prompt them with a list, though

Reply:
I initially thought I'd use the arg to be more consistent with the deploy command, but I was considering converting them all to flags since I don't think users would want the tag used anyway. It would be both annoying and unnecessary to have to scale by tag

Reply:
We should probably remove the tag entirely since I think it just causes confusion. Like what happens if you apply to version A but then push version B. Both should be scaled to the same size you would assume.

Reply:
You can't currently deploy multiple components of different tags to the same environment anyway, so that's another reason not to use the tag